docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -11,56 +11,61 @@ tags:
 ---
 
 A List Filter is the SQL-like string that is provided as the parameter to an [Advanced Visibility](/concepts/what-is-advanced-visibility) List API.
+List Filter [Search Attribute](/concepts/what-is-a-search-attribute) names are case sensitive, and each List Filter is scoped by a single [Namespace](/concepts/what-is-a-namespace).
 
-- [How to use a List Filter using tctl](/tctl-v1/workflow#list)
+A List Filter that uses a time range has a resolution of 1 ns on Elasticsearch 7+ and 1 µs for SQL.
 
-The following is an example List Filter:
+### Supported operators
 
-```
-WorkflowType = "main.YourWorkflowDefinition" and ExecutionStatus != "Running" and (StartTime > "2021-06-07T16:46:34.236-08:00" or CloseTime > "2021-06-07T16:46:34-08:00") order by StartTime desc
-```
+:::note
+
+Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
+
+:::
+
+A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute) names, Search Attribute values, and the following supported operators:
 
-[More example List Filters](#example-list-filters)
+- **=, !=, >, >=, <, <=**
+- **AND, OR, ()**
+- **BETWEEN ... AND**
+- **IN**
+- **ORDER BY**
 
-A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute) names, Search Attribute values, and Operators.
+The **ORDER BY** operator is supported only when Elasticsearch is used as the Visibility store.
```

Comment:
```suggestion
The **ORDER BY** operator is supported only when Elasticsearch is used as the Visibility store.
Additionally, custom Search Attributes of the `Text` type cannot be used in **ORDER BY** clauses.
```

Reply:
I just think it reads better and makes more sense to group the "ORDER BY" information - don't you?